src/index.js

```diff
@@ -8,8 +9,23 @@ const defaultOptions = {
   source: true,
 };
 
+const defaultMtrcConf = {
+  h1: H1,
+  h2: H2,
+  h3: H3,
+  h4: H4,
+  h5: H5,
+  h6: H6,
+  code: Code,
+  p: P,
+  a: A,
+  li: LI,
+  ul: UL,
+};
+
 export default {
-  addWithInfo(storyName, info, storyFn, _options) {
+  addWithInfo(storyName, info, storyFn, _options, _mtrcConf) {
```

Comment:
Hi @rook91 

There's one small bug in the code. **No need to fix this because we can avoid it**

It's not quite clear but the `info` parameter in this line is optional. Users can give the story function as the second argument, options as the third, etc. So the mtrcConf argument is not always the fourth argument. It could be either the third or the fourth.

As we already have an options object, we don't need to add another argument. `mtrcConf` can be a field in the options object. Try something like this:

```js
const mtrcConf = { ...defaultMtrcConf };
if (_options && _options.mtrcConf) {
  Object.assign(mtrcConf, _options.mtrcConf);
}
```

I'll merge this as soon as it's fixed.